helpers/GenerateResourcesAndImage.ps1

```diff
@@ -102,10 +111,16 @@ Function GenerateResourcesAndImage {
         [Parameter(Mandatory = $False)]
         [string] $GithubFeedToken,
         [Parameter(Mandatory = $False)]
+        [string] $AzureClientId,
+        [Parameter(Mandatory = $False)]
+        [string] $AzureClientSecret,
+        [Parameter(Mandatory = $False)]
+        [string] $AzureTenantId,
+        [Parameter(Mandatory = $False)]
         [Switch] $Force
     )
 
-    if (([string]::IsNullOrEmpty($GithubFeedToken)))
```

Comment:
Removed the unnecessary parentheses; thanks @miketimofeev for pointing the issue out!